lib/tools.js

```diff
@@ -351,6 +358,20 @@ function stringify (obj) {
   }
 }
 
+function Formatters (level, bindings, log) {
```

Comment:
Does this need to be a constructor function?

Reply:
agreed - unless there's some measurably significant reason not to instead return an object then let's just return an object